### DISCUSSION TITLE
目前哪个版本 稳定、基本功能都有？

### DISCUSSION THREAD

**Discussion by yeguobin8** [ttttupup/wxhelper#374] [General]
用的朋友推荐一下。自己打算先用起来，自己再改进学习，大家推荐哪个版本？我看wxhelper-3.9.2.26-v10，这版本功能功能较完善，是否推荐？

**Comment by ghost** [comment]
能下得到3.9.2.26版本再说吧。
各个版本都有点点优势，都有短板。感觉上下的一致性比较差。各自为战了。

**Comment by jamesliu668** [comment]
大家谁有3.9.8.25的64位版本的安装包么？求群友分享一个，谢谢啦

**Reply by xufa-he** [reply]
https://github.com/tom-snow/wechat-windows-versions/releases
